docs/community/governance.md

```diff
@@ -0,0 +1,106 @@
+
+### 3. Advisors
+
+> Members who have been an Executive before and are still on campus. Late third year onwards.
+
+Advisors, in general, have more technical experience. They actively suggest better technologies to be used in KOSS. They also make sure the Executive Heads are not over-prioritizing anything and are relaxed. They continuously take updates from the Executive Heads about what is going in KOSS. They intervene when asked by an Executive to resolve any conflict. They can also lead meetings if Executive Heads do not object.
+
+One of the Advisors acts as the Treasurer of KOSS and oversees the finances. See [Accounts](/docs/community/accounts).
+
+### 4. Alumni
+
+“Members who have graduated from KGP”
+
+The role has no formal duties, as of now.
+
+
+## Guidelines
+
+### Guidelines for Executive Heads -
+
+1. Do not miss a meeting unless extremely required (health or out of station). Your lack of involvement will psychologically disolve your responsibilities. Schedule meetings according to the availability of all the Executive Heads.
+1. Be in sync among yourselves. Physically meet at-least once a week. Slack and Emails do not take importance over your coordination. You will build an amazing friendship with this, which will remain beyond KOSS.
+1. Respect time of your fellow Executives and Juniors. Do not slack off.
+1. Always have a lump-sum timeline for events and internal workshops for a particular semester (or year). Core Team members are an essential part of overall execution, so not having a timeline puts them at blindside, and they cannot manage their time.
+1. Your Core Team Members are your KGP juniors as well. Make sure they prioritize their mental and physical health.
+1. Follow "Survival first" rule. You need to ensure that your Core Team members have learnt enough to continue in KOSS. You can prioritize this over conducting a fancy workshop.
+1. Make sure KOSS is a sexually and emotionally non-hostile place to work. Importance of this point is definitely not determined by where it is placed.
+1. Build strong connections with your Core Team members. Have regular 1:1s. Prefer Meetings and Calls over Slack and Email.
+1. Keep your Executive Members as involved as possible.
+
+### Guidelines for Executive Members -
+
+1. Give Executive Heads their freedom. Do not interrupt or hinder their momentum of doing things.
+1. Offer suggestions whenever you feel like it.
+1. Come to meetings. Build relations with Core Team Members. Be active on Slack.
+
+### Guidelines for Core Team Members -
+
+1. Take initiatives and ownership of KOSS.
+1. Make sure you are learning things and making good friends in KOSS, else you will definitely lose interest and leave.
+1. Practice public speaking and teaching as often as possible.
+1. There will always be people better than you in one or more aspect, it applies to your batch too. Don’t get intimidated by this. Remember, you can never know all the things.
```

Comment:
I think we can restructure some of the guidelines mentioned here.

1. We believe in self-motivation and a sense of ownership. Try to take initiatives and tasks on your own.
2. Make sure you are active within the community. Reach out to your batchmates and seniors once in a while. Discuss stuff with people, or you might end up losing interest.
3. Communication skills are vital for any line of work. Practice public speaking and teaching as often as possible. Try to get over your stage fear (if you have one).
4. There will always be people who do certain things better than you in one or more aspect. Don’t get intimidated by this. Instead, try to get the most of out it. Knowledge-sharing is how we as individuals grow.

Reply:
Done.